README.md

```diff
@@ -154,6 +155,24 @@ $ hs-airdrop -h
 Note that if you ran `hs-airdrop` before mainnet, you will need to upgrade to
 the latest version of hs-airdrop and clear the cache (`rm -rf ~/.hs-tree-data`).
 
+You can verify the data being signed will actually be sending you HNS coins to the address you specified by following these [instructions](https://github.com/handshake-org/hs-airdrop/issues/36).
+
+### Debugging
+If you receive an error message similar to this:
+```
+Attempting to create proof.
+This may take a bit.
+Decrypting nonce...
+Downloading: https://github.com/handshake-org/hs-tree-data/raw/master/nonces/111.bin...
+Error: Client network socket disconnected before secure TLS connection was established
+at TLSSocket.onConnectEnd (_tls_wrap.js:1084:19)
+at Object.onceWrapper (events.js:273:13)
+at TLSSocket.emit (events.js:187:15)
+at endReadableNT (_stream_readable.js:1085:12)
+at process._tickCallback (internal/process/next_tick.js:63:19)
+```
+You should download https://github.com/handshake-org/hs-tree-data to your home directory at `~/.hs-tree-data` and re-run your command.
```

Comment:
This is not totally right, if you still have testnet tree data the error is a checksum mismatch: https://github.com/handshake-org/hs-airdrop/issues/38#issuecomment-586728569

I think because the tree data files are so large that sometimes GitHub just doesn't serve them or fails somehow. This issue: https://github.com/handshake-org/hs-airdrop/issues/57 I think the user just retried without making any adjustments, and it worked.

Reply:
Ok, so remove that section? If it's not a catch all answer then probably shouldn't be in README